spec/yaml/swiftnav/sbp/navigation.yaml

```diff
@@ -213,7 +213,7 @@ definitions:
             type: u16
             units: mm
             desc: |
-              Position accuracy estimate. 
```

Comment:
if we are going to change this, lets up units in the estimate description as well [mm] so it matches the other descriptions above.